src/backend/parser/cypher_expr.c

```diff
@@ -251,14 +251,11 @@ static Node *transform_A_Const(cypher_parsestate *cpstate, A_Const *ac)
         d = boolean_to_agtype(boolVal(&ac->val));
         break;
     default:
-        if (ac->isnull)
-        {
+        if (ac->isnull) {
```

Comment:
Braces do *not* go at the end of the line of code, they go on the following line.,